lib/portlayer/exec/exec.go

```diff
@@ -155,18 +159,123 @@ func eventCallback(ie events.Event) {
 						container.onStop()
 					}
 					log.Debugf("Container(%s) state set to %s via event activity",
-						container.ExecConfig.ID, newState.String())
+						container, newState.String())
 					// regardless of update success failure publish the container event
 					publishContainerEvent(container.ExecConfig.ID, ie.Created(), ie.String())
 				}()
 			case StateRemoved:
-				log.Debugf("Container(%s) %s via event activity", container.ExecConfig.ID, newState.String())
+				log.Debugf("Container(%s) %s via event activity", container, newState.String())
 				Containers.Remove(container.ExecConfig.ID)
 				publishContainerEvent(container.ExecConfig.ID, ie.Created(), ie.String())
 			}
+		} else {
+			switch ie.String() {
+			case events.ContainerRelocated:
```

Comment:
why this event is handled in else statement? I assume portlayer will not put container into relocated status, and then it's not possible to be same to container current state